docs/what-is-hiptensor.rst

```diff
@@ -10,7 +10,7 @@ What is hipTensor?
 
 hipTensor is a work-in-progress (WIP) high-performance HIP library for tensor primitives. It is AMD's C++ library for accelerating tensor primitives which can leverage specialized GPU matrix cores on AMD's latest discrete GPUs. hipTensor is currently powered by the composable kernel library. The API is designed to be portable with the Nvidia cuTensor library, allowing those users to easily migrate to the AMD platform.
```

Comment:
I think this topic might need to be removed. 
It looks like the same content is on the landing page (index.rst)

Reply:
Don't see it here:
![image](https://github.com/user-attachments/assets/418e17e4-20ff-4568-b3d5-3ffb5b2e283a)

Reply:
@CongMa13 Good to remove

Reply:
removed